.gitignore

```diff
@@ -61,3 +61,4 @@ typings/
 .next
 .DS_Store
 dist/
+package-lock.json
```

Comment:
Please add a new line at the end of file.

Reply:
CSS files do not require new lines at the end of the file. Error is thrown by github when one is applied. Check how eslint handles this @Ogutu-Brian.